test/e2e/performanceprofile/functests/1_performance/irqbalance.go

```diff
@@ -318,14 +318,14 @@ func getIrqBalanceBannedCPUs(node *corev1.Node) (cpuset.CPUSet, error) {
 	cmd := []string{"cat", "/rootfs/etc/sysconfig/irqbalance"}
 	conf, err := nodes.ExecCommandOnNode(cmd, node)
 	if err != nil {
-		return cpuset.NewCPUSet(), err
+		return cpuset.New(), err
```

Comment:
Is there a migration guide (for upgrading k8s versions) that states the move from cpuset.NewCPUSet() to cpuset.New() ?  just as a reference if there is one

Reply:
I followed the changes in the original code:
https://github.com/kubernetes/kubernetes/commit/a0c989b99a5857d7e211d64e8cd3f09da0dd9ee1

Reply:
This is the commit history: https://github.com/kubernetes/kubernetes/commits/v1.27.4/pkg/kubelet/cm/cpuset/cpuset.go

Reply:
also: https://github.com/kubernetes/kubernetes/pull/116761

Reply:
I the package migration  planned to 1.28

Reply:
right, 1.27 had prep work only IIRC